build.sh

```diff
@@ -124,13 +124,13 @@ install_dotnet_cli()
         __dotnetclipath=${__cli_dir}
     fi
 
-    if [ ! -f "${__cli_dir}/bin/dotnet" ]; then
+    if [ ! -f "${__cli_dir}/dotnet" ]; then
         echo "CLI could not be installed or not present."
         exit 1
     fi
 
-    echo "cat ${__cli_dir}/.version"
-    cat "${__cli_dir}/.version"
+    echo "ls ${__cli_dir}/sdk"
```

Comment:
Ah that. I updated it to print out the version of what's there now after invoking Install-Cli.ps1